proxy-node-gateway/src/main/java/uk/gov/ida/notification/GatewayApplication.java

```diff
@@ -154,23 +144,16 @@ private void registerExceptionMappers(Environment environment) {
     private void registerResources(GatewayConfiguration configuration, Environment environment) throws Exception {
         SamlFormViewBuilder samlFormViewBuilder = new SamlFormViewBuilder();
 
-        HubAuthnRequestGenerator hubAuthnRequestGenerator = createHubAuthnRequestGenerator(configuration);
-
-        EidasAuthnRequestValidator eidasAuthnRequestValidator = createEidasAuthnRequestValidator(configuration, connectorMetadataResolverBundle);
         HubResponseValidator hubResponseValidator = createHubResponseValidator(configuration);
 
-        SamlRequestSignatureValidator samlRequestSignatureValidator = createSamlRequestSignatureValidator(connectorMetadataResolverBundle);
-
         EidasSamlParserService eidasSamlParserService = configuration
             .getEidasSamlParserServiceConfiguration()
             .buildEidasSamlParserService(environment);
 
         environment.jersey().register(new EidasAuthnRequestResource(
-                configuration,
-                hubAuthnRequestGenerator,
-                samlFormViewBuilder,
-                eidasAuthnRequestValidator,
-                samlRequestSignatureValidator));
+                eidasSamlParserService,
+                new VSPService(),
```

Comment:
I understand this will be replaced with the new VerifyServiceProviderProxy @Wynndow ?

Reply:
Correct, there's a PR up at the moment do do just that.